assets/js/components/common/Downlink.jsx

```diff
@@ -30,6 +33,7 @@ class Downlink extends Component {
     position: "last",
     showRefresh: false,
     queue: [],
+    region: "undefined",
```

Comment:
maybe change this string to "default" instead of "undefined"? kinda weird since undefined is an actual entity in js

Reply:
I had "default" originally but looks like router is expecting undefined: https://discord.com/channels/404106811252408320/966788666913611867/969664709445967952 but I can confirm with Pierre

Reply:
addressed in latest comit